### PR TITLE
Donation widget/quick option

### PR DESCRIPTION
**Pull request by duranmla**
**What:**
Allow user to quick select predefined amount values

**Why:**
relates to Development of Donation widget for upcoming release

**How:**
- Add new `QuickOption` component within the donation widget
- Integrate new component into Donation Once and Monthly forms

> Disclaimer: it may be some complexity that we can improve/remove over the next iterations of the widget. Nevertheless, bare in mind that the layout of `QuickOption` has
been done trying to facilitate the iteration over the options without breaking the way all the options are place.

**Media:**
image

https://www.loom.com/share/ce0fd7ac65ab4176baa4d514e53c936f

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ module.exports = {
         sans: ['Roboto', 'Helvetica', 'Arial', 'sans-serif']
       },
       fontSize: {
+        tiny: '0.6rem',
```
Comment (orlando) [inline tailwind.config.js:57]: should we keep the same pattern as the others? xxs for example

**Review by orlando — COMMENTED** [review]
just a small comment

**Comment by orlando** [comment]
Also, if we can limit the amount to 50k that would be great, just like this
image

**Comment by duranmla** [comment]
Do we have some link for the contact us? I wonder if it is convenient to just set a maximum on the input number or we actually want something else.

**Review by arathjz — APPROVED** [review]

**Comment by duranmla** [comment]
I've addressed the feedback @orlando 
image

_The call to action appears while typing no on submit as it was easy for me to implement and IMO is better to alert the user as soon as possible._

**Pull request merged**